docs/src/pages/lab/speed-dial/speed-dial.md

```diff
@@ -21,3 +21,11 @@ You can provide an alternate icon for the closed and open states using the `icon
 of the `SpeedDialIcon` component.
 
 {{"demo": "pages/lab/speed-dial/OpenIconSpeedDial.js"}}
+
+## Persistant action tooltips
+
+The SpeedDialActions tooltips can be be displayed persistently so that users don't have to long-press in order to see the tooltip on touch devices.
+
+It is enabled here across all devices for demo purposes, but in production could use the isTouch logic to conditionally set the property.
```

Comment:
```diff
- but in production could use the isTouch logic
+ but in production it could use the `isTouch` logic
```

Not sure about the typo but the formatting change would be nice.

Reply:
Not really a typo, but yours is grammatically clearer. Agree on the formatting.